src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate from;
+        LocalDate to;
+        from = parseDate(dateFrom);
+        to = parseDate(dateTo);
```

Comment:
```suggestion
        LocalDate from  = parseDate(dateFrom);
        LocalDate to = parseDate(dateTo);
```
тут не потрібно розділяти оголошення і декларацію змінних